javascript/src/test/components/ChannelMessages/MessageScrollable.test.js

```diff
@@ -209,19 +226,60 @@ describe("MessageScrollableView tests", () => {
             "subtype": "channel_join",
             "ts": "1594143066.000200",
             "user": "U017218J9B3",
-            "text": "<!channel> This is an announcement",
+            "text": "Call me at <tel:+01234567890|+0 123 456 7890>",
             "channel": "section-6pm",
             "user_profile": {
                 "real_name": "Test Person"
             }
         }
         const {getByText} = render(<MessageScrollableView messages={[exampleMessage]}/>);
-        setTimeout(function (){
-            var bracketElement = getByText(/<!channel>/);
-            expect(bracketElement).toHaveLength(0);
-            bracketElement = getByText(/!channel/);
-            expect(bracketElement).toBeInTheDocument();
-        }, 500)
+        const linkElement = getByText("+0 123 456 7890");
+        expect(linkElement.href).toEqual("tel:+01234567890");
+        
+    });
+
+    test("Channel links are clickable", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = {
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "Please post in <#C01K1CR63MX|help-jpa02>",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            }
+        }
+        const {getByText} = render(<MessageScrollableView messages={[exampleMessage]}/>);
+        const linkElement = getByText(/help-jpa02/);
+        expect(linkElement.href).toEqual("http://localhost/member/channels/help-jpa02");
```

Comment:
I thought Prof conrad wants the channel links to go to the slack archives link like so:

https://ucsb-cs156-w21.slack.com/archives/C01K1CR63MX

Did he say that he wanted this to direct to the app page? If he did disregard this. Sorry might be out of the loop.

Reply:
![image](https://user-images.githubusercontent.com/59621442/110039574-0c53e580-7d07-11eb-95b5-ebe07f8b6b7b.png)

I think he did want us to do that at first but then changed his mind.